target/mips/cpu.c

```diff
@@ -32,6 +32,133 @@
 #include "exec/exec-all.h"
 #include "qemu/error-report.h"
 
+#ifndef TARGET_MIPS64
```

Comment:
If you can be bothered, using
```
const char mips_gp_regnames_flat[32][5] = {
    {"zero"}, {"at"}, {"v0"}, {"v1"}, {"a0"}, {"a1"},{"a2"}, {"a3"},
...
};
```
Reduces the binary size (and avoids relocations for PIE binaries)